lua/autorun/server/sv_focus.lua

```diff
@@ -0,0 +1,26 @@
+local tabbedOutPlys = {}
+
+util.AddNetworkString( "CFC_AttentionMonitor_gameHasFocus" )
+util.AddNetworkString( "CFC_AttentionMonitor_sendData" )
+
+hook.Add( "PlayerDisconnected", "CFC_AttentionMonitor_playerleave", function(ply)

```

Comment:
nit
```suggestion
hook.Add( "PlayerDisconnected", "CFC_AttentionMonitor_PlayerLeave", function(ply)
```
alot of your hook/net names forget to capitalize the last word